src/postgresql_gpd.tf

```diff
@@ -0,0 +1,105 @@
+# review postgreql:
+# naming and resource group
+# avaibility zones, backup redundancy
+
+resource "azurerm_resource_group" "flex_data" {
+  count = var.env_short != "d" ? 1 : 0
+
+  name     = format("%s-pgres-flex-rg", local.project)
```

Comment:
good advice, for the moment I'ld not change , but in the next PR we will do so. Also because we have to solve the problem we have with 2 app services in the same plan and the vnet integration